frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -590,17 +590,20 @@ function ReaderPaging:getBookLocation()
     if ctx then
         -- We need a copy, as we're getting references to
         -- objects ReaderPaging/ReaderView may still modify
-        local current_location = {}
-        for i=1, #ctx do
-            current_location[i] = util.tableDeepCopy(ctx[i])
-        end
+        local current_location = util.tableDeepCopy(ctx)
         return current_location
     end
 end
 
 function ReaderPaging:onRestoreBookLocation(saved_location)
+    if (not saved_location) or (not saved_location[1]) then
```

Comment:
(You don't need the parens I think - no time to go look at Lua operator precedence - but it feels we do that elsewhere without any.)

Reply:
I put them in, as it is easier to tell right away without having to lookup the manual every time we look at the code :)

Reply:
We also strive for some consistency - having different ways just because _you_ are not confident with Lua operators precedence (neither am I) feels odd, and will create more differences when people grep for how we usually do it (which is just what I did to suggest we don't need the parens :)).